web/test/pages/onboarding.test.tsx

```diff
@@ -44,6 +44,41 @@ describe("onboarding form", () => {
     useMockRouter({});
   });
 
+  it("changes url pathname every time a user goes to a different page", async () => {
+    subject = render(
+      <Onboarding displayContent={createEmptyOnboardingDisplayContent()} municipalities={[]} />
+    );
+    expect(subject.getByTestId("step-1"));
```

Comment:
for all these expects in all the tests, it looks like we're not expecting anything?
should have a `.toBeInTheDocument()` or something like that on them all